Add ember-svg-jar writeup for DecEmber 12 (#421)
ember-learn/ember-blog: source/2019-12-12-countown-to-the-new-year-ember-svg-jar.md

---
title: Countdown to The New Year - Ember SVG Jar
author: Seema Shariat, Amy Lam
tags: 2019, Addons, DecEmber
alias: 2019-12-12-countdown-to-the-new-year-ember-svg-jar
responsive: true
---

> **This is the twelth in our December series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**

Reviewer: ```suggestion
**This is the twelfth in our December series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
```

## Day 12

Today we're highlighting [ember-svg-jar](https://www.emberobserver.com/addons/ember-svg-jar). 

READMORE

### What It Does

> There's a reason ember-svg-jar has a 10 out of 10 on Ember Observer! ember-svg-jar enables you to use SVG images in Ember apps effortlessly.

Reviewer: ```suggestion
There's a reason `ember-svg-jar` has a 10 out of 10 on Ember Observer! The `ember-svg-jar` addon enables you to effortlessly use SVG images in Ember apps.
```

### Why I Like It

> A common theme in the Ember community is zero configuration. In that vein, with ember-svg-jar, you get a visual workflow out of the box to search and preview icons in one place. Check out the [live demo](https://svgjar-demo.web.app/) and empower your team with this feature!

Reviewer: ```suggestion
A common theme in the Ember community is zero configuration. In that vein, with `ember-svg-jar`, you get a visual workflow out of the box to search and preview icons in one place. Check out the [live demo](https://svgjar-demo.web.app/) and empower your team with this feature!
```

> ember-svg-jar helps you reduce the size of your assests. Using [svgo](https://github.com/svg/svgo), SVGs are automatically optimized, which can cut your file size by half or more. 

Reviewer: ```suggestion
The `ember-svg-jar` addon helps you reduce the size of your assets. Using [svgo](https://github.com/svg/svgo), SVGs are automatically optimized, which can cut your file size by half or more. So useful!
```

<!-- alex ignore easy-->

> The addon is flexible and extensible. ember-svg-jar provides an easy to use helper `{{svg-jar "asset-name"}}`, support for both `inline` and `symbol` embedding methods, and the ability to copy SVG as CSS background.

Reviewer: ```suggestion
The addon is flexible and extensible. ember-svg-jar provides an easy-to-use helper `{{svg-jar "asset-name"}}`, support for both `inline` and `symbol` embedding methods, and the ability to copy SVG as CSS background.
```

adding hyphens to adjective.
